packages/taquito/src/operations/operations.ts

```diff
@@ -168,11 +172,11 @@ export class Operation {
         .pipe(
           switchMap(() => this.polling$),
           switchMap(() => this.currentHead$),
-          filter(head => head.header.level - this._foundAt >= conf - 1),
+          filter(head => head.header.level - this._foundAt >= conf! - 1),
```

Comment:
I think it is because of the change from Required<Config> to Partial<Config> in the context class. Maybe using Partial there is not the right thing to do, could we use something else?

Reply:
I have introduced "throw errors" in this case as suggested.